docs/proposals/202105256-cluster-class-and-managed-topologies.md

```diff
@@ -229,8 +262,182 @@ type LocalObjectTemplate struct {
 }
 ```
 
+**VariablesClass**
+
+```golang
+// VariablesClass defines the variables which can be configured
+// in the Cluster topology and used in patches.
+type VariablesClass struct {
+  // Definitions define the variables inline.
+  Definitions []VariableDefinitionClass `json:"definitions,omitempty"`
+}
+ 
+// VariableDefinitionClass defines a variable which can 
+// be configured in the Cluster topology and used in patches.
+type VariableDefinitionClass struct {
```

Comment:
A default value could also be useful here, especially when variables are used in conjunction with patches. Especially since arrays can only be modified using `add`.

Reply:
Nevermind, that's done using the Schema.